third-party/libfabric/Makefile

```diff
@@ -6,12 +6,12 @@ CHPL_MAKE_HOST_TARGET = --target
 include $(CHPL_MAKE_HOME)/make/Makefile.base
 
 #
-# Cray X* builds are cross-compilations.
+# Cray builds are cross-compilations.
 #
-ifneq (, $(filter cray-x% cray-shasta,$(CHPL_MAKE_TARGET_PLATFORM)))
-CHPL_LIBFABRIC_CFG_OPTIONS += --host=$(shell uname -m)-cle-linux-gnu
-else ifneq ($(CHPL_MAKE_HOST_PLATFORM),$(CHPL_MAKE_TARGET_PLATFORM))
-CHPL_LIBFABRIC_CFG_OPTIONS += --host=$(CHPL_MAKE_TARGET_PLATFORM)-unknown-linux-gnu
+ifneq (, $(filter cray-x% hpe-cray-ex,$(CHPL_MAKE_TARGET_PLATFORM)))
```

Comment:
Move hpe-cray-ex to be with cray-cs (think it's right in all 3p except libfab)

Reply:
Oops -- thanks!  Fixed this.